### DISCUSSION TITLE
Why are the assembly results directly looped into multiple

### DISCUSSION THREAD

**Discussion by zhaoying888** [Kinggerm/GetOrganelle#154] [Q&A]
![A4LC~O(J$8YNJ_{1{51CY%V](https://user-images.githubusercontent.com/97380840/156185499-72cdf489-e7fc-4027-9bcf-203ae991f654.png)
The assembly results are directly looped into multiple, how should be selected to compare and annotate with reference
[Uploading get_org.log.txt…]()

**Comment by Kinggerm** [comment]
Please refer to https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#why-are-there-so-many-path_sequencefasta-files

Check the assembly graph carefully and confirm the structure.

**Comment by zhaoying888** [comment]
Roger that, thank you! I will read and revise the web page carefully


| |
昆明植物所 赵颖
|
|
***@***.***
|
签名由网易邮箱大师定制
On 3/24/2022 11:21，Jianjun ***@***.***> wrote：

Please refer to https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#why-are-there-so-many-path_sequencefasta-files
